serverless.yml

```diff
@@ -55,12 +55,12 @@ resources:
               Effect: Allow
               Action: s3:GetObject
               Resource: arn:aws:s3:::${self:custom.s3BucketName}/*
-              Principal:
-                AWS:
-                  Fn::Join:
-                    - " "
-                    - - "arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity"
-                      - !Ref OriginAccessIdentity
+              Principal: "*"
```

Comment:
Should we return back OAI configuration? Principal: "*" means that your bucket is gonna be public and available for anonymous. Limiting access by OAI gives access to a specific identity and closes bucket from the outside. https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-bucket-user-policy-specifying-principal-intro.html